arcgis-ios-sdk-samples/Edit data/Offline edit and sync/OfflineEditing.storyboard

```diff
@@ -1,11 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It seems like nothing changed in the storyboard. Am I correct? 😳

Reply:
Yes, whatever is different is just because I opened the file.

Reply:
In the future, if you accidentally open a storyboard (or nib) and Xcode makes changes to it, please don't commit those changes.